packages/node_modules/@ciscospark/redux-module-mercury/src/index.js

```diff
@@ -0,0 +1,51 @@
+import {Map} from 'immutable';
+
+export const SET_CONNECTING = `mercury/SET_CONNECTING`;
+export const SET_DISCONNECTING = `mercury/SET_DISCONNECTING`;
+export const SET_CONNECTED = `mercury/SET_CONNECTED`;
+export const SET_DISCONNECTED = `mercury/SET_DISCONNECTED`;
```

Comment:
Doesn't look like these disconnect action names aren't being used anywhere? Are these gonna be used when everything gets migrated over?